File: test_connect.pl
#!/opt/local/bin/perl
use IO::Socket;
my $sock = new IO::Socket::INET (
				PeerAddr => 'www.google.com',
				PeerPort => '80',
				Proto => 'tcp',
				);
die "No me puedo conectar: $!\n" unless $sock;
print "------------------\n";
print "Conexion exitosa !\n";
print "------------------\n";
$sock->send("GET / HTTP1.0\n\n");
$sock->recv($text,128000);
print "Recibiendo:\n", $text,"'\n";
close($sock);

File: test_connect.py
#!/usr/bin/python
import socket

s = socket.socket()
host = 'www.google.com'
port = 80

s.connect((host, port))

msg = 'GET / HTTP1.0\n\n'
s.send(msg)

buf = s.recv(1024)
while buf:
    print buf
    buf = s.recv(1024)
      
s.close

